packages/enzyme-context-apollo-3/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "enzyme-context-apollo-3",
+  "version": "3.0.0",
```

Comment:
We should start at `1.0.0` since this is a new package!

Reply:
Done!